aries-site/src/pages/components/table.mdx

```diff
@@ -0,0 +1,240 @@
+import { Anchor, Box, Image } from 'grommet';
+import { Example } from '../../layouts';
+import { ExampleImagePreview } from '../../components';
+import {
+  TableAnatomy,
+  TableExample,
+  TableFixedHeaderExample,
+  TableMultiSelectExample,
+  TableSingleSelectExample,
+  TableSortable,
+  TableSummaryExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/table/TableExample.js"
+  docs="https://v2.grommet.io/datatable?theme=hpe#props"
+  figma="https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table-Component?node-id=195%3A0"
+  height={{ min: 'small' }}
+>
+  <TableExample />
+</Example>
+
+## What makes up a table?
+Tables enable users to accomplish a task by finding records of interest, then take action against them.
+They present structured data in a grid, with columns and rows to aid readability, scanability, and show 
+the meaning of the data.
+
+<TableAnatomy />
+
+**Controls**<br />
+Table controls should live above the table and facilitate a user's ability to locate and 
+take actions against records of interest. A controls section may contain [search and filter 
+controls](#searching-and-filtering-tables), as well as [action controls](#selecting-multiple-records-and-batch-actions) 
+such as batch applying settings or data export.
+
+**Table header**<br />
+Contain table header cells which concisely label the column's contents.
+Column headers may be enhanced by enabling [sorting](#sorting).
+
+**Table body**<br />
+Consists of rows which allow users scan, read and take action on the data. 
+Rows may be enhanced by enabling actions such as [record selection](#selecting-single-records) and 
+navigation to record detail.
```

Comment:
```suggestion
navigation to a detail page for that record.
```

Reply:
modified to read "navigation to a record's detailed view."